### PR TITLE
fix : active page menu style [#76 ]

### PR DESCRIPTION
**Pull request by imdhabib**
Resolved : #76 
Screenshot (624)

### CODE REVIEW

File: src/components/PrimaryNavItems.tsx [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
     <>
-      {navigationItems.map((navigationItem) => (
+      {navigationItems.map((navigationItem, index) => (
         <li key={navigationItem.title}>
           <Link href={navigationItem.href!}>
-            <span className="hover:text-sky-500 dark:hover:text-sky-400">
+            <span
+              onClick={() => setActive(index)}
+              className={`${
+                active === index ? 'text-sky-500' : 'none'
```
Comment (sakilk130) [inline src/components/PrimaryNavItems.tsx:20]: remove none class 
` className={`${active === index && 'text-sky-500'}  hover:text-sky-500 dark:hover:text-sky-400`}`
OR
`className={clsx(active === index && 'text-sky-500','hover:text-sky-500 dark:hover:text-sky-400' )}`

**Review by sakilk130 — CHANGES_REQUESTED** [review]
image

As you can see in the above picture, a class has printed the name "none", so the logic must be changed.

File: src/components/PrimaryNavItems.tsx (hunk 1 above)
Comment (mimukit) [reply src/components/PrimaryNavItems.tsx:20]: Use `clsx` for conditional className.

File: src/components/Navigation.tsx [hunk 2]
```diff
@@ -33,6 +33,7 @@ export const Navigation: React.FC<NavigationProps> = ({
                           {
                             'font-semibold text-sky-500 before:bg-sky-500':
                               link.href === router.pathname,
+
```
Comment (mimukit) [inline src/components/Navigation.tsx:36]: Remove unnecessary extra space on this file.

**Review by mimukit — CHANGES_REQUESTED** [review]

File: src/components/PrimaryNavItems.tsx (hunk 1 above)
Comment (imdhabib) [reply src/components/PrimaryNavItems.tsx]: Done

**Pull request merged**